### DISCUSSION TITLE
Finally got a method to remove items from Win-X in Win11 22H2

### DISCUSSION THREAD

**Discussion by vippium** [valinet/ExplorerPatcher#2134] [Show and tell]
After searching for long time, I got a method to remove items from the Win-X menu in 22H2 builds. As all know that there is no way to edit the Win-X menu after updating to 22H2. Even Win-X menu Editor also don't work in latest builds.
But after researching on this, I got a method to remove items. I will try to check a method for adding our own shortcuts to this (as we can do in Win10). But currently check for removing items.

There are 2 folders i.e. "Current User" and for "All Users" :

For Current User:
**C:\Users\Administrator\AppData\Local\Microsoft\Windows\WinX**

For All Users:
**C:\Users\Default\AppData\Local\Microsoft\Windows\WinX**

![2023-06-06_08h17_32](https://github.com/valinet/ExplorerPatcher/assets/93650504/87489803-c238-43b9-9219-70c855fe39bc)

In this, I've removed Powershell Admin and Task Manager as I don't need those shortcuts.
In this, MS merged Powershell and wt together, so, removing both wt also remove both Powershell
 I'm trying to add cmd in replacement of Powershell. I'll update this after I found a way to add items too.

**Comment by pyrates999** [comment]
Is this windows 10 or 11?

**Comment by vippium** [comment]
Only for Win11... As it is now very difficult in Win11

**Comment by CorruptionVR** [comment]
is there a way to use this to add items?

**Comment by vippium** [comment]
> is there a way to use this to add items?

Not found till yet...I'm searching for it. But got a method to edit names and move items to new section.
The older method adds items in the menu, but doesn't show in the menu. And replacing the classic shortcuts with the settings options will only work till u don't restart. After restart, the options from menu vanishes.
I think that microsoft is forcing Win-X to use Settings apps only.

I'll update the SS as soon as I found method.

**Comment by pyrates999** [comment]
Please update your original post and subject to indicate this is only for windows 11 and not windows 10.

**Comment by vippium** [comment]
After a week, I have tried my best but no luck regarding adding new items in Win-X menu. I've find the registry keys of Win-X items but they are not working in Win11 latest version. But I got a method to replace the Powershell with Command Prompt. This is not a proper way to replace it but ya..you can replace it.

Just make a shortcut of cmd from System32 and remane it as Windows PowerShell and replace it with original powershell in **C:\Users\Administrator\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Windows PowerShell\**. And rename it using Win-X menu Editor by Winaero.

This is the only way to replace pwrshell with cmd. But the issue with it is that the name you got on title bar after launching the cmd was Windows PowerShell, not Command Prompt. If you have no issue with it, then go ahead.

**Reply by iamjonny** [reply]
Did you see the information provided by James or Rafael, and address the "hash" factor in shortcut name creation?
https://james-rankin.com/articles/customizing-the-winx-menu-and-making-it-user-specific/
Just arriving here keen to config explorer to allow access to common tools, thought I'd look at winx menu. There's also hotkeys and quicklaunch in my list of explorer-projects to ...explore

**Reply by peasoft** [reply]
It's title bar is now Command Prompt!

1. Copy `%AppData%\Microsoft\Windows\Start Menu\Programs\System Tools\Command Prompt.lnk` to `%AppData%\Microsoft\Windows\Start Menu\Programs\Windows PowerShell\` using explorer then rename it to `Windows PowerShell`
2. The `Command Prompt.lnk`'s name is **localized with `desktop.ini`**, so you will get a `desktop.ini` in your `Windows PowerShell` folder
3. Use notepad to open the hidden `desktop.ini`, replace `Command Prompt.lnk` to `Windows PowerShell.lnk`
4. Now the `Windows PowerShell.lnk`'s **display name, or the localized name** becomes `Command Prompt`

**Reply by pyrates999** [reply]
That's because you're now running cmd.exe instead of powershell.exe.  What is this supposed to accomplish?

**Reply by peasoft** [reply]
> But the issue with it is that the name you got on title bar after launching the cmd was Windows PowerShell, not Command Prompt.

The window's title is same as the shortcut file itself, not the target.

**Comment by Amrsatrio** [comment]
FYI: I'm now reimplementing Windows 10 1904x's Win+X menu. I hope this would make it customizable again using other tools.

**Reply by vippium** [reply]
Great to see it
